Cubical/Structures/Group/Base.agda

```diff
@@ -31,8 +33,15 @@ record IsGroup {G : Type ℓ}
   invr : (x : G) → x + (- x) ≡ 0g
   invr x = inverse x .fst
 
-record Group : Type (ℓ-suc ℓ) where
+η-isGroup : {G : Type ℓ} {0g 0g' : G} {_+_ _+'_  : G → G → G} { -_ -'_  : G → G}
+         → 0g ≡ 0g'
+         → _+_ ≡ _+'_
+         → -_ ≡ -'_
+         → IsGroup 0g _+_ -_ ≡ IsGroup 0g' _+'_ -'_
+η-isGroup id1 id2 id3 i = IsGroup (id1 i) (id2 i) (id3 i)
 
+record Group : Type (ℓ-suc ℓ) where
+  no-eta-equality
```

Comment:
Do you really need this no-eta-equality? I would have thought it's enough to have it for IsGroup as that contains the proofs

Reply:
I do actually. Addition runs through the Freudenthal suspension theorem, so there are some pretty heavy proofs there too.

Reply:
I see. We should take a close look at this together. I'm very suspicious that something needs to be optimized in the definition of the operations.